cloud/client.go

```diff
@@ -972,6 +986,105 @@ func (c *client) WhoAmI() (string, string, bool, error) {
 	return email, authType, writeAccess, nil
 }
 
+func (c *client) LaunchSatellite(name, orgID string) (*SatelliteInstance, error) {
+	req := pipelinesapi.LaunchSatelliteRequest{
+		OrgId:    orgID,
+		Name:     name,
+		Platform: "linux/amd64", // TODO support arm64 as well
+	}
+	status, body, err := c.doCall("POST", "/api/v0/satellites", withAuth(), withJSONBody(&req))
+	if err != nil {
+		return nil, err
+	}
+	if status != http.StatusOK {
+		return nil, errors.Errorf("failed launching satellite: %s", body)
+	}
+	var resp pipelinesapi.LaunchSatelliteResponse
+	err = c.jm.Unmarshal(bytes.NewReader([]byte(body)), &resp)
+	if err != nil {
+		return nil, errors.Wrap(err, "failed to unmarshal LaunchSatellite response")
+	}
+	return &SatelliteInstance{
+		Name:     name,
+		Status:   resp.Status.String(),
+		Version:  resp.Version,
+		Platform: "linux/amd64",
+	}, nil
+}
+
+func (c *client) GetOrgID(orgName string) (string, error) {
+	orgs, err := c.ListOrgs()
+	if err != nil {
+		return "", err
+	}
+	for _, o := range orgs {
+		if o.Name == orgName {
+			return o.ID, nil
+		}
+	}
+	return "", errors.Errorf("org not found: %s", orgName)
+}
+
+func (c *client) ListSatellites(orgID string) ([]SatelliteInstance, error) {
+	url := fmt.Sprintf("/api/v0/satellites?orgId=%s", orgID)
```

Comment:
nit: probably not needed, but makes it more robust:
```suggestion
	url := fmt.Sprintf("/api/v0/satellites?orgId=%s", url.QueryEscape(orgID))
```

(there's a few other spots in this code that could use this too.